testing/scripts/playwright_test.py

```
@@ -1946,6 +1946,286 @@ async def test_api_v220_rate_limit_contract(page: Page) -> None:
+        total is not None and "," in (total or ""),
+        f"got {total!r}"
+    )
+    assert_eq("total IPs value is 65,536", total, "65,536")
+
+
+async def test_eslint_clean(page: Page) -> None:  # noqa: ARG001
+    """Run ESLint and assert exit code 0."""
+    import subprocess
+    section("ESLint — app.js clean")
+    result = subprocess.run(
+        ["npm", "run", "lint:js"],
```

Ruff (0.15.9)

[error] Starting a process with a partial executable path

(S607)